### ISSUE TITLE
do not show all the answers

### ISSUE THREAD

**Issue by Heldwiin** [Iku/Google-Forms-to-Discord#112]
Hello,
Is it possible to show only certain responses on Discord rather than all responses?
Thank you

**Comment by Kelo** [comment]
Yes. You can use an if statement in the code to look for specific attributes of an answer.
